inspektor/lint.py

```diff
@@ -64,6 +64,7 @@ def get_opts(self):
             if self.args.enable:
                 pylint_args.append('--enable=%s' % self.enabled_errors)
             pylint_args.append('--reports=no')
+            pylint_args.append('--errors-only')
```

Comment:
OK, looking at the help more carefully, I found the option. It's `--score=no`. That'll be fixed in the next iteration.